internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json

```diff
@@ -25683,58 +25683,6 @@
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP585",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP245",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP585",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-MM_SSP585",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP245",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP585",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP245",
-        "MaxTemp_Daily_1PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_EnsembleAcrossModels_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_EnsembleAcrossModels_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_GFDL-CM4_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_GFDL-CM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_GFDL-ESM4_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_GFDL-ESM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_HADGEM3-GC31-LL_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_HADGEM3-GC31-LL_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_HADGEM3-GC31-MM_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_MPI-ESM1-2-HR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_MPI-ESM1-2-HR_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_MPI-ESM1-2-LR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayADecade_CMIP6_MPI-ESM1-2-LR_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_EnsembleAcrossModels_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_EnsembleAcrossModels_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-CM4_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-CM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-MM_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP585",
```

Comment:
Are these deletions expected?

Reply:
This might be - @jehangiramjad

Reply:
Yes, these should be expected. None of them in the path /resolved_mcf/raw_stat_vars/cmip6/vars.mcf